zerver/lib/actions.py

```diff
@@ -4079,6 +4083,17 @@ def email_not_system_bot(email: Text) -> None:
     if is_cross_realm_bot_email(email):
         raise ValidationError('%s is an email address reserved for system bots' % (email,))
 
+def make_unique_display_name(target_realm: Realm, full_name: Text) -> Text:
+    full_name_counter = 1
+    full_name_initial = full_name
+    realm_users = UserProfile.objects.filter(realm=target_realm).values('full_name')
+    normalized_full_names = [unicodedata.normalize('NFKC', item) for sublist in
+                             [list(k.values()) for k in realm_users] for item in sublist]
```

Comment:
I think `realm_users` is already a flat list, not a dict of lists, so the 2-layer nesting is not necessary. Evidence: https://github.com/zulip/zulip/blob/master/zerver/tests/test_messages.py#L2266-L2267.

Reply:
Agree. Simplified things a bit. Thanks @rht !